### ISSUE TITLE
Styling issue: Comments with code blocks extend past container

### ISSUE THREAD

**Issue by rohitpaulk** [codecrafters-io/frontend#1591]
From https://app.codecrafters.io/courses/http-server/stages/1: 

<img width="1049" alt="Screenshot 2024-04-12 at 14 25 05" src="https://github.com/codecrafters-io/frontend/assets/3893573/0605f4b5-13d9-49fd-8ba3-1073223f58b2">

(These comments might not be visible, so you'll need to create your own to actually replicate the issue)

**Comment by rohitpaulk** [comment]
@VasylMarchuk can you confirm whether this has been fixed? I remember a seeing a PR that addressed something similar.

**Comment by VasylMarchuk** [comment]
Nope, not yet fixed, tried looking at it today but didn't find an obvious fix

**Comment by VasylMarchuk** [comment]
This has something to do with how avatars are displayed in the comments: if I hide them all, the sizing works flawlessly
